README.md

```diff
@@ -66,98 +69,73 @@ For guidance on developing for PowerToys, please read the [developer docs](/doc/
-- Our submodules will no longer auto fetch to prevent locking issues. If you want a refresher on how to do this, head to [our dev docs](https://github.com/microsoft/PowerToys/tree/main/doc/devdocs#get-submodules-to-compile)
-- Localization system shifted to Touchdown from CDPx.  This should remove many of the loc issues.
-- Consolidated a lot of the naming of EXEs and DLLs along with projects
-- Update to spell checker.  Thanks @jsoref
-- /dup response has been added
-- /reportbug /bugreport will ask for a "report bug" zip
+#### Video Conference Mute
+- Fixed crashing bug with Zoom and other clients. We found someone we could remotely debug with and identify the actual crashing part.
+- Change of behavior: When leaving a meeting, VCM will now leave your microphone in the state it was.  This mimics behavior of applications if VCM was not present.
+- Change of behavior: When you exit PowerToys, your current microphone state will remain.
 
 #### Community contributions
 
 We'd like to directly mention certain contributors (in alphabetical order) for their continued community support this month and helping directly make PowerToys a better piece of software. 
 
-[@Aaron-Junker](https://github.com/Aaron-Junker), [@ChaseKnowlden](https://github.com/ChaseKnowlden), [@CleanCodeDeveloper](https://github.com/CleanCodeDeveloper), [@cyberrex5](https://github.com/cyberrex5), [@davidegiacometti](https://github.com/davidegiacometti), [@franky920920](https://github.com/franky920920), [@gsuberland](https://github.com/gsuberland),  [@jay-o-way](https://github.com/jay-o-way), [@jsoref](https://github.com/jsoref), [@niels9001](https://github.com/niels9001), and [@ricardosantos9521](https://github.com/ricardosantos9521)
+[@Aaron-Junker](https://github.com/Aaron-Junker), [@bdoserror](https://github.com/bdoserror), [@CleanCodeDeveloper](https://github.com/CleanCodeDeveloper), [@cyberrex5](https://github.com/cyberrex5), [@davidegiacometti](https://github.com/davidegiacometti), [@edwinzap](https://github.com/edwinzap), [@franky920920](https://github.com/franky920920),  [@jay-o-way](https://github.com/jay-o-way), [@jsoref](https://github.com/jsoref), [@niels9001](https://github.com/niels9001), and [@ricardosantos9521](https://github.com/ricardosantos9521)
+
 
-#### What is being planned for v0.55
+#### What is being planned for v0.56
 
-For [v0.55][github-next-release-work], we'll work on adding more stability in with VCM and getting dev file preview pane added in so we get 150 file types :)
+For [v0.56][github-next-release-work], TODO
 
-- We are working to heavily reduce / remove the UAC prompt over the next few releases on install. This is a big shift so it is spanning multiple releases so we can isolate issues if they do occur. Work is tracked in [#10126](https://github.com/microsoft/PowerToys/issues/10126)
-- Getting the dev file preview pane work integrated. (Monaco Editor)
 - .NET 6 upgrade to all available surfaces
```

Comment:
```suggestion
- .NET 6 upgrade to all available surfaces
- Fix to improve search speed in Windows Terminal plugin
- Hopefully we can provide you new features for Window Walker that allows you to close a window and to kill the processes of a window.
```

- Terminal fix: I plan to create the PR this weekend.
- Window Walker features: I plan to create the PR for the next release.

Reply:
Let's write your suggestion